### ISSUE TITLE
optionalDependencies are not installed

### ISSUE THREAD

**Issue by wayneashleyberry** [yarnpkg/yarn#1001]
**Do you want to request a *feature* or report a *bug*?**

Bug

**What is the current behavior?**

Optional dependencies are not installed.

**If the current behavior is a bug, please provide the steps to reproduce.**

```sh
$ yarn add gulp-imagemin
$ cd node_modules
$ ls -l | grep imagemin
drwxr-xr-x   7 Wayne  staff  238 Oct 13 16:31 gulp-imagemin
drwxr-xr-x   7 Wayne  staff  238 Oct 13 16:31 imagemin
```

**What is the expected behavior?**

Optional dependencies mentioned in [package.json](https://github.com/sindresorhus/gulp-imagemin/blob/cb12e3332dce75dd581a907bb80fa8ce4e61c183/package.json#L52-L57) should be installed, as per NPM.

```sh
$ npm install --save gulp-imagemin
$ cd node_modules
$ ls -l | grep imagemin
drwxr-xr-x   6 Wayne  staff  204 Oct 13 16:32 gulp-imagemin
drwxr-xr-x   6 Wayne  staff  204 Oct 13 16:32 imagemin
drwxr-xr-x   6 Wayne  staff  204 Oct 13 16:32 imagemin-gifsicle
drwxr-xr-x   6 Wayne  staff  204 Oct 13 16:32 imagemin-jpegtran
drwxr-xr-x   6 Wayne  staff  204 Oct 13 16:32 imagemin-optipng
drwxr-xr-x   6 Wayne  staff  204 Oct 13 16:32 imagemin-svgo
```

**Please mention your node.js, yarn and operating system version.**

- node v6.8.0
- macOS Sierra
- yarn v0.15.1

**Comment by glenjamin** [comment]
See https://github.com/yarnpkg/yarn/pull/789 for more info

TLDR is this is fixed on master.

**Comment by wayneashleyberry** [comment]
awesome, thanks @glenjamin
